File: Service_eHospital_DongNai_A/Implement/DM_MaDuocService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class DM_MaDuocService : BaseService<DM_MaDuoc, IDM_MaDuocRepository>
    , IDM_MaDuocService
    {
    private readonly IDM_MaDuocRepository _DM_MaDuocRepository;
    public DM_MaDuocService(IDM_MaDuocRepository DM_MaDuocRepository) : base(DM_MaDuocRepository)
    {
    _DM_MaDuocRepository = DM_MaDuocRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A/DM_MaDuoc.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DM_MaDuoc } from './DM_MaDuoc.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DM_MaDuocService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaDuoc'];

    List: DM_MaDuoc[] | undefined;
    ListFilter: DM_MaDuoc[] | undefined;
    FormData!: DM_MaDuoc;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DM_MaDuoc";
    }
}

